src/main/java/amazin/controller/ApplicationController.java

```diff
@@ -1,12 +1,16 @@
 package amazin.controller;
 
+import amazin.model.ShoppingCart;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Controller;
 import org.springframework.ui.Model;
 import org.springframework.web.bind.annotation.GetMapping;
 
 import amazin.repository.BookRepository;
 
+import java.util.Iterator;
+import java.util.Optional;
+
```

Comment:
unused imports

Reply:
no changes made in this file except for these 3 different imports which were not used